java-engine/src/main/java/io/getunleash/engine/UnleashEngine.java

```diff
@@ -34,20 +36,66 @@
 
 public class UnleashEngine {
 
-  private Instance instance;
+  private static Instance instance;
+  private static ExportFunction alloc;
+  private static ExportFunction dealloc;
+  private static ExportFunction checkEnabled;
+  private static ExportFunction checkVariant;
+  private static ExportFunction getMetrics;
+  private static ExportFunction deallocResponseBuffer;
+  private static ExportFunction getLogBufferPtr;
+  private static ExportFunction listKnownToggles;
+  private static ExportFunction getCoreVersion;
+  private static ExportFunction getBuiltInStrategies;
+  private static ExportFunction newEngine;
   private long enginePointer;
-  private ExportFunction alloc;
-  private ExportFunction dealloc;
-  private ExportFunction checkEnabled;
-  private ExportFunction checkVariant;
-  private ExportFunction getMetrics;
-  private ExportFunction deallocResponseBuffer;
-  private ExportFunction getLogBufferPtr;
-  private ExportFunction listKnownToggles;
   private Memory memory;
-
   private final CustomStrategiesEvaluator customStrategiesEvaluator;
 
+  static {
+    ImportValues imports =
```

Comment:
Ripped out of the constructor. This gives us a path to accessing WASM functions without having an UnleashEngine object, which is nice for static data like the version